pkg/cmd/image/list.go

```diff
@@ -181,17 +180,15 @@ type imagePrintable struct {
 	Platform string // nerdctl extension
 }
 
-func printImages(ctx context.Context, options types.ImageListCommandOptions, client *containerd.Client, imageList []images.Image) error {
+func printImages(ctx context.Context, client *containerd.Client, imageList []images.Image, options types.ImageListOptions) error {
 	digestsFlag := options.Digests
-	var w io.Writer
-	w = os.Stdout
 	if options.Format == "wide" {
 		digestsFlag = true
 	}
 	var tmpl *template.Template
 	switch options.Format {
 	case "", "table", "wide":
-		w = tabwriter.NewWriter(w, 4, 8, 4, ' ', 0)
+		options.Stdout = tabwriter.NewWriter(options.Stdout, 4, 8, 4, ' ', 0)
```

Comment:
nit: I feel we should keep and update the `w` variable (e.g., line 186-187) and use `w` here&below, instead of persisting the change in `options.Stdout`.

Reply:
Sure, fixed.